hyperemitter.js

```diff
@@ -21,12 +21,20 @@ var defaults = {
   reconnectTimeout: 1000
 }
 
-function HyperEmitter (db, schema, opts) {
+function HyperEmitter (db, codecs, opts) {
   if (!(this instanceof HyperEmitter)) {
-    return new HyperEmitter(db, schema, opts)
+    return new HyperEmitter(db, codecs, opts)
+  }
+
+  this.codecs = codecs
+  this.messages = this.codecs
+  this.status = new EventEmitter()
+
+  if (Buffer.isBuffer(this.codecs)) {
+    this.status.emit('buffers are being depreciated')
```

Comment:
@mcollina What should I do with a string? Is that an acceptable input to Protobuf?